tests/ChatTests/Groups.hs

```diff
@@ -141,6 +141,11 @@ chatGroupTests = do
     it "member contact is deleted silently, then considered disabled" testMembershipProfileUpdateContactDisabled
     it "profile update without change is ignored" testMembershipProfileUpdateNoChangeIgnored
     it "change of profile contact link is ignored" testMembershipProfileUpdateContactLinkIgnored
+  describe "block member for all" $ do
+    it "messages are marked as moderated" testBlockForAllModerated
+    it "messages are fully deleted" testBlockForAllFullDelete
+    it "another admin can unblock" testBlockForAllAnotherAdminUnblocks
+    it "can't repeat block/unblock" testBlockForAllCantRepeat
```

Comment:
does it test for members blocked before new member joins?

Reply:
no, I've missed this case